ToolBox/ForexVolumeDownloader/ForexVolumeWriter.cs

```diff
@@ -0,0 +1,84 @@
+﻿using QuantConnect.Data.Custom;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using QuantConnect.Data;
+using QuantConnect.Util;
+
+namespace QuantConnect.ToolBox.FxVolumeDownloader
+{
+    public class ForexVolumeWriter
+    {
+        private readonly Symbol _symbol;
+        private readonly string _market;
+        private readonly string _dataDirectory;
+        private readonly Resolution _resolution;
+        private readonly string _folderPath;
+
+        public ForexVolumeWriter(Resolution resolution, Symbol symbol, string dataDirectory)
+        {
+            _symbol = symbol;
+            _resolution = resolution;
+            _dataDirectory = dataDirectory;
+            _market = _symbol.ID.Market;
+            _folderPath = Path.Combine(new[] {_dataDirectory, "base", _market.ToLower(), _resolution.ToString().ToLower()});
+            if (resolution == Resolution.Minute)
```

Comment:
If the resolution is unsupported (minute or second) you might considering throwing and `ArgumentException`